docs/production/security-model.md

```diff
@@ -244,9 +244,20 @@ strength allowed is controlled by two settings in
   messages when a message contains a link to an image.  You can
   control this using the `INLINE_IMAGE_PREVIEW` setting.
 
-* A Zulip server can make outgoing HTTP requests through features like
-  outgoing webhooks and embedded video previews. End users have
-  (limited) control the content of these HTTP requests. As a result,
+* Zulip may make outgoing HTTP connections to other servers in a
+  number of cases:
+
+  * Outgoing webhook bots (creation of which can be restricted)
+  * Inline images in messages
+  * Previews of Dropbox image links
+  * Inline image previews (must be configured to be enabled)
```

Comment:
Inline image previews appears to be a duplicate of "inline images in messages".  And I think the dropbox feature should be classified as a special case of "Image links".  We could do a follow-up pass of making these links all links to the corresponding feature in /help/, but this is definitely an improvement.